Board Election and Governance Blog Post (#1296)
tauri-apps/tauri-docs: blog/2023-06-15-tauri-board-elections-and-governance-updates.mdx

---
authors: [beanow, lorenzolewis]
description: The Tauri team is happy to announce the 1.3.0 release.
---

# Tauri Board Elections & Governance Update

The Tauri Programme is celebrating it's two year anniversary of Tauri becoming a programme within [The Commons Conservancy](https://commonsconservancy.org/) as well as the one year milestone of the [Tauri 1.0 release](/blog/2022/06/19/tauri-1-0). This also means we're preparing for the next round of Tauri Board Director elections to welcome in the next chapter of Tauri.

We wanted to celebrate these milestones but also use this as a chance to reflect on how we've grown since the Tauri 1.0 launch and how to continue evolving Tauri. The Tauri Working Group has been building the next iteration of the governance model for the past several months to make it more accessible for new and existing contributors, and ultimately sustainable for the future of Tauri.

<!-- truncate -->

## Board Elections

> The Tauri Board of Directors is the central decision-making body for the Tauri Programme and is responsible for the overall health and stability of Tauri the Tauri Programme. Additional details can be found on the [Tauri Governance page](/about/governance).

Reviewer: ```suggestion
The Tauri Board of Directors is the central decision-making body for the Tauri Programme and is responsible for the overall health and stability of the Tauri Programme. Additional details can be found on the [Tauri Governance page](/about/governance).
```

> In order to provide continuity from one year to another, the elections of seats are staggared over the course of 2 years. This means that once year a portion of the seats are elected, and the following year the remaining seats are elected. A Tauri Director seat is an elected positions with a terms of 2 years. The Board has a minimum of 3 Directors and a maximum of 7. This year there are 5 seats open for election.

Reviewer: ```suggestion
In order to provide continuity from one year to another, the elections of seats are staggered over the course of 2 years. This means that once year a portion of the seats are elected, and the following year the remaining seats are elected. A Tauri Director seat is an elected position with a term of 2 years. The Board has a minimum of 3 Directors and a maximum of 7. This year there are 5 seats open for election.
```

> In order to provide continuity from one year to another, the elections of seats are staggered over the course of 2 years. This means that once year a portion of the seats are elected, and the following year the remaining seats are elected. A Tauri Director seat is an elected position with a term of 2 years. The Board has a minimum of 3 Directors and a maximum of 7. This year there are 5 seats open for election.

Reviewer 2: ```suggestion
In order to provide continuity from one year to another, the elections of seats are staggered over the course of 2 years. This means that one year a portion of the seats are elected, and the following year the remaining seats are elected. A Tauri Director seat is an elected position with a term of 2 years. The Board has a minimum of 3 Directors and a maximum of 7. This year there are 5 seats open for election.
```

2. Prepare a written introduction about yourself that covers who you are, your history and relevance for Tauri, and what you would like to bring to the Board.
3. **Apply prior to July 2nd, 2023** by emailing [board@tauri.app](mailto:board@tauri.app), messaging the `@board` role on the [Tauri Discord](https://discord.com/invite/tauri), or by directly messaging Daniel Thompson-Yvetot on Discord (`@Denjell`) or email ([denjell@tauri.app](mailto:denjell@tauri.app)).

Voting will take place starting on July 5th through July 12th, 2023 where Tauri Working Group Members will cast their votes. We're expecting to announce the results on or before July 16, 2023. If elected, we'll ask you to sign [this pledge](https://dracc.commonsconservancy.org/0016/) if you haven't done so previously.

If you have any questions or would like more information please reach out by emailing [board@tauri.app](mailto:board@tauri.app), messaging the `@board` role on the [Tauri Discord](https://discord.com/invite/tauri), or by directly messaging Daniel Thompson-Yvetot on Discord (`@Denjell`) or email ([denjell@tauri.app](mailto:denjell@tauri.app)).

---

## Governance Update

> Tauri has always had a Working Group that's responsible for the day-to-day direction and execution, but as we've grew it became clear that we needed to evolve the Working Group. We also wanted to give new Working Group members a clear places to jump in and get started. We thought of how we could achieve this while building and iterating on the existing framework.  **With these goals in mind, we created two new concepts within the Working Group: Domains and Domain Leads.**

Reviewer 2: ```suggestion
Tauri has always had a Working Group that's responsible for the day-to-day direction and execution, but as we grew it became clear that we needed to evolve the Working Group. We also wanted to give new Working Group members a clear places to jump in and get started. We thought of how we could achieve this while building and iterating on the existing framework.  **With these goals in mind, we created two new concepts within the Working Group: Domains and Domain Leads.**
```

### Domains

A Domain in the Working Group represents a specific area of interest. We're starting with these 4 domains:

- **Development**: Developing and maintaining the Tauri software
- **Community**: Looks after the broader community resources and the public presence of Tauri
- **Governance & Guidance**: Involved with all things organizational in nature
- **Operations**: Responsible for all the tooling and infrastructure Tauri needs to get work done

> This gives clear entry points for a new member of the Working Group to jump into right away. It also allows us to continue growing our core competency of core Tauri development while giving additional focus to all areas which support the Tauri organization as a whole. Each of these Domains will be made up of 

Reviewer: ```suggestion
This gives clear entry points for a new member of the Working Group to jump into right away. It also allows us to continue growing our core competency of core Tauri development while giving additional focus to all areas which support the Tauri organization as a whole. Each of these domains will be represented by Domain Members and Domain Leads, but membership is not mutually exclusive.
```

### Domain Leads

Within a domain, there is a position for 2-3 Domain Leads. These are core individuals in the Working Group that are critical to their respective Domain. They help by setting the direction and priorities for their domain while also serving as key leaders for the Working Group as a whole.

> Leads are elected twice a year, in the spring and in the fall. Since this governance update is being rolled out mid-term we're appointing interim leads that have been key members of the Working Group. This fall we'll hold the first round of elections to formally elect the Domain Leads. We'll announce information for that process when we're closer to election season.

Reviewer 2: ```suggestion
Leads are appointed twice a year, in the spring and in the fall. Since this governance update is being rolled out mid-term we're appointing interim leads that have been key members of the Working Group. This fall we'll reflect on the initial experience and host a new round to appoint the Domain Leads. We'll announce information for that process when we're closer to "appointment season".
```

Important detail this. The Board appoints Domain Leads. So calling that *elected* could suggest that it's the WG doing the electing. Appointment is also the term that is used in regulation, which is leading in this regard.

Also slight rewording to try and express what that round would look like in fall. Primarily a reflection moment, a call for new applicants and another "term" for appointments.

(They're not formalized terms, but soft terms)

---

> We hope that these changes provide the foundation to continue the journey the Tauri Working Group has been on while also  catalyzing the continued growth and sustainability of the organization. You can learn more about these updates in the [Governance and Guidance repo on GitHub](https://github.com/tauri-apps/governance-and-guidance). Give us your feedback on the [GitHub Discussion for this post](https://github.com/tauri-apps/tauri-docs/discussions/1116).

Reviewer 2: Just a heads up the Discussion link goes to a random unrelated PR atm :laughing:

Author: Yeah, this _should_ be the next one that get's created if I can do numbers right ;)

Reviewer 2: Nah it isn't because the number is shared between issues and PRs too. So this redirected to #1116

> We hope that these changes provide the foundation to continue the journey the Tauri Working Group has been on while also  catalyzing the continued growth and sustainability of the organization. You can learn more about these updates in the [Governance and Guidance repo on GitHub](https://github.com/tauri-apps/governance-and-guidance). Give us your feedback on the [GitHub Discussion for this post](https://github.com/tauri-apps/tauri-docs/discussions/1116).

Author: ```suggestion
We hope that these changes provide the foundation to continue the journey the Tauri Working Group has been on while also  catalyzing the continued growth and sustainability of the organization. You can learn more about these updates in the [Governance and Guidance repo on GitHub](https://github.com/tauri-apps/governance-and-guidance). Give us your feedback on the [GitHub Discussion for this post](https://github.com/tauri-apps/tauri-docs/discussions/1297).
```
